config/rootcompute/kube-1.24/clusterrole-kubernetes-maximal-permission-policy.yaml

```diff
@@ -26,3 +26,6 @@ rules:
 - apiGroups: ["networking.k8s.io"]
   resources: ["ingresses/status"]
   verbs: ["get", "list", "watch"]
+- apiGroups: [""]
+  resources: ["pods", "pods/status", "pods/log", "pods/exec", "pods/attach", "pods/portforward", "pods/proxy", "pods/ephemeralcontainers" ]
```

Comment:
We have to look into whether some of those sub-resources need more than `get` / `list` / `watch` actions in order to be useful for the workload user (thinking of things like `ephemeralcontainers`)

Reply:
Good point! Let me double-check that.

Reply:
Updated! please have another look!